src/navigator/IFrameNavigator.ts

```diff
@@ -2216,6 +2216,15 @@ export class IFrameNavigator extends EventEmitter implements Navigator {
       ttsModule.speakPlay();
     }
   }
+  startReadAloudBySegment(startTime: number, endTime: number) {
+    if (
+      this.rights.enableMediaOverlays &&
+      this.mediaOverlayModule !== undefined &&
+      this.hasMediaOverlays
+    ) {
+      this.mediaOverlayModule?.startReadAloudBySegment(startTime, endTime);
+    }
+  }
```

Comment:
Here I think we can do one check directly for `this.mediaOverlayModule.startReadAloudBySegment` so we don't have to use the optional function call `?.`
```suggestion
  startReadAloudBySegment(startTime: number, endTime: number) {
    if (
      this.rights.enableMediaOverlays &&
      this.mediaOverlayModule.startReadAloudBySegment != null &&
      this.hasMediaOverlays
    ) {
      this.mediaOverlayModule.startReadAloudBySegment(startTime, endTime);
    }
  }
```